iphone/Classes/TiUIListView.m

```diff
@@ -1762,7 +1762,7 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
     }
     if (TiDimensionIsDip(height)) {
         return height.value;
-    } else if (TiDimensionIsAutoSize(height)) {
+    } else if (TiDimensionIsAutoSize(height) || TiDimensionIsUndefined(height)) {
```

Comment:
While setting Ti.UI.auto for row height , it gives undefined. Rather I found that for Ti.UI.Fill check similar condition should applied . Updated check with TiDimensionIsAutoFill(height). 
Thanks.